<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompaniesDialog</class>
 <widget class="QDialog" name="CompaniesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Companies</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Properties:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="cmdNewCompany">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>New Company</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdSaveChanges">
         <property name="text">
          <string>Save Changes</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdDeleteCompany">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Delete Company</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="trvCompanies">
       <property name="maximumSize">
        <size>
         <width>255</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Existing Companies</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QTreeWidget" name="trvProperties">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>6</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Property Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>No. Of Units</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Code:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="txtCode"/>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="txtEmail"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="txtPhysicalAddress"/>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="txtCompanyName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Physical Address:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Postal Address:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Company Name:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="txtMobileNo"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mobile:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="txtPostalAddress"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="txtContactPerson"/>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLineEdit" name="txtTelNo"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Tel:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Contact Person:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmdNewCompany</tabstop>
  <tabstop>cmdSaveChanges</tabstop>
  <tabstop>cmdDeleteCompany</tabstop>
  <tabstop>trvCompanies</tabstop>
  <tabstop>txtCompanyName</tabstop>
  <tabstop>txtCode</tabstop>
  <tabstop>txtPhysicalAddress</tabstop>
  <tabstop>txtPostalAddress</tabstop>
  <tabstop>txtContactPerson</tabstop>
  <tabstop>txtMobileNo</tabstop>
  <tabstop>txtTelNo</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>trvProperties</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
